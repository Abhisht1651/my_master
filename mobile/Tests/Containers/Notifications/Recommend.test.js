// @flow
import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import Recommend from '../../../App/Components/Notifications/Recommend'

describe('Recommend', () => {
  it('Recommend component renders correctly', () => {
    const tree = shallow(<Recommend />)
    expect(tree).toMatchSnapshot()
  })

  it('should render the Text Component', () => {
    const wrapper = shallow(<Recommend />)
    expect(wrapper.find('Text')).toHaveLength(4)
  })

  it('should render the Image Component', () => {
    const wrapper = shallow(<Recommend />)
    expect(wrapper.find('Image')).toHaveLength(2)
  })
})
