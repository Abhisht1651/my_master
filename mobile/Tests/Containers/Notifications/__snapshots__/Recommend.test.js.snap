// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recommend Recommend component renders correctly 1`] = `
<View
  style={
    Object {
      "zIndex": 9,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#F7F7F7",
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          Object {
            "width": 132,
          }
        }
      >
        <View
          style={
            Object {
              "paddingLeft": 60,
              "paddingTop": 42.714285714285715,
            }
          }
        >
          <Image
            source={
              Object {
                "default": "",
              }
            }
            style={
              Object {
                "height": 27.928571428571427,
                "width": 36,
              }
            }
          />
        </View>
      </View>
      <View
        style={
          Object {
            "width": 574,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View
            style={
              Object {
                "paddingTop": 37.785714285714285,
                "width": 464,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#1F1F60",
                  "fontFamily": "BananaGrotesk-Light",
                  "fontSize": 23,
                  "lineHeight": 31.21428571428571,
                }
              }
            >
              <Text
                style={
                  Object {
                    "fontFamily": "BananaGrotesk-Bold",
                  }
                }
              >
                Recommended:
              </Text>
               
              Donald Trump
            </Text>
          </View>
          <View
            style={
              Object {
                "paddingRight": 28.000000000000004,
                "paddingTop": 34.5,
              }
            }
          >
            <ForwardRef
              activeOpacity={1}
              onPress={[Function]}
            >
              <Image
                source={
                  Object {
                    "default": "",
                  }
                }
                style={
                  Object {
                    "height": 4.928571428571428,
                    "width": 30,
                  }
                }
              />
            </ForwardRef>
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View
            style={
              Object {
                "paddingTop": 4.928571428571428,
                "width": 464,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#8F8FAF",
                  "fontFamily": "BananaGrotesk-Regular",
                  "fontSize": 19.71428571428571,
                  "lineHeight": 23,
                }
              }
            >
              34K
               people talking about this
            </Text>
          </View>
          <View
            style={
              Object {
                "paddingRight": 28.000000000000004,
                "paddingTop": 4.928571428571428,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#8F8FAF",
                  "fontFamily": "BananaGrotesk-Regular",
                  "fontSize": 19.71428571428571,
                  "lineHeight": 23,
                }
              }
            >
              90 min
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingTop": 34.5,
        }
      }
    >
      <View
        style={
          Object {
            "paddingLeft": 53.99999999999999,
            "paddingRight": 46,
          }
        }
      >
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
